app/src/styles.sass

```diff
@@ -48,6 +50,7 @@ input
   justify-content: center
   margin-top: 1em
   margin-bottom: 1em
+  margin-left: - $medium-space
```

Comment:
It works, but it would be much better to not have a left margin on the button instead of having the left border of the div not aligned with the other elements.